app/src/main/java/com/sdp13epfl2021/projmag/model/Message.kt

```diff
@@ -4,6 +4,39 @@ package com.sdp13epfl2021.projmag.model
  * createdAt parameter is the time in milliseconds check Date()
  */
 
-data class Message(val messageContent: String, val sender : ImmutableProfile, val createdAt : Long) {
-
+data class Message(val messageContent: String, val userId: String, val createdAt : Long) {
+    companion object {
+        object FieldNames {
+            const val MESSAGE_CONTENT = "message"
+            const val SENDER = "sender"
+            const val CREATION_DATE = "creationDate"
+        }
+        private const val MAX_MESSAGE_SIZE = 200
+        /**
+         * Function that creates a Message
+         * returns a message wrapped in a success wrapper, or a failure with the explanation wrapped as a string
+         *
+         * @param messageContent : content of the message
+         * @param sender : profile of the user that send the message
```

Comment:
The KDoc need to be updated.
```suggestion
         * @param userId the id of the user that send the message
```